packages/functions/test/waitUntil.test.ts

```diff
@@ -26,15 +24,23 @@ test.each([null, undefined, {}])(
   'waitUntil does not throw an error when context is %s',
   input => {
     const promise = Promise.resolve();
-    globalThis[Symbol.for('@vercel/request-context')] = input;
+    globalThis[
+      // @ts-ignore
+      Symbol.for(
+        '@vercel/request-context'
+      ) as unknown as keyof typeof globalThis
+    ] = input;
     expect(() => waitUntil(promise)).not.toThrow();
   }
 );
 
 test('waitUntil calls ctx.waitUntil when available', async () => {
   const promise = Promise.resolve();
   const waitUntilMock = vi.fn().mockReturnValue(promise);
-  globalThis[Symbol.for('@vercel/request-context')] = {
+  globalThis[
+    // @ts-ignore
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏can you add to the comment what/why we're ignoring?

Reply:
it would probably be better to `// @ts-expect-error` instead of ignoring so if the error isn't happening anymore we will be prompted to remove this line